src/d3a/models/area/redis_dispatcher/market_event_dispatcher.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, area, root_dispatcher, redis):
             MarketEvent.TRADE.value: Event(),
             MarketEvent.OFFER.value: Event(),
             MarketEvent.OFFER_DELETED.value: Event(),
-            MarketEvent.OFFER_CHANGED.value: Event(),
+            MarketEvent.OFFER_SPLIT.value: Event(),
```

Comment:
Irrelevant to this PR, but since I remembered it, it would be great to replace the hardcoded construction of child_response_events member with a dict comprehension.